### PR TITLE
online tab + comb validation + test

### PR DESCRIPTION
**Pull request by OdedAnswer**

### CODE REVIEW

File: server-side/services/filter-rule.service.ts [hunk 1]
```diff
@@ -155,6 +158,12 @@ export class FilterRuleService extends BasicTableService<FilterRule>{
                     Filter: basicFilterRule.Key,
                     PermissionSet: "Sync"
                 });
+                filterRules.push({
```
Comment (Chasky-H) [inline server-side/services/filter-rule.service.ts:161]: I'm not sure we should be doing this.
especialy for admins.
Please talk to Eyal about this.

File: server-side/installation.ts [hunk 2]
```diff
@@ -42,9 +42,10 @@ export async function uninstall(client: Client, request: Request): Promise<any>
 export async function upgrade(client: Client, request: Request): Promise<any> {
     try {
         console.log(`upgrade from version ${request.body.FromVersion} to version ${request.body.ToVersion}`);
-        // if we are upgrading from version < 0.0.55, we need to update the FilterRules permissionSet
-        //if (request.body.FromVersion && semver.compare(request.body.FromVersion, '0.0.55') < 0) {
+        // if we are upgrading from version < 0.0.56, we need to update the FilterRules permissionSet and schema
```
Comment (Chasky-H) [inline server-side/installation.ts:45]: try fixing this by using the exacy same verison of semver we are using in other addons (I think Nebula uses it)

File: server-side/services/filter-rule.service.ts (hunk 1 above)
Comment (OdedAnswer) [reply server-side/services/filter-rule.service.ts:161]: I asked him yesterday if I should create the same defaults for Online as well and he said yes

File: server-side/installation.ts (hunk 2 above)
Comment (OdedAnswer) [reply server-side/installation.ts:45]: it is the same version exactly (^7.3.8)

**Review by Chasky-H — APPROVED** [review]

**Pull request merged**